src/main/scala/com/clevercloud/warp10client/Runner.scala

```diff
@@ -104,56 +104,54 @@ object Runner {
             GTS(
               classname = `class`,
               labels = labels,
-              points = (series \\ "v").map { seriesContentArrays => // [[point_1], [point_2], ...]
+              points = (series \\ "v").flatMap { seriesContentArrays => // [[point_1], [point_2], ...]
                 seriesContentArrays.asArray.get.map { point => // [point_i]
-                  point.asArray.get match { // [timestamp, lat, lon, elev, value] is point's content
-                    case Vector(timestamp: Json, value: Json) => {
+                  point.asArray.getOrElse(Vector(0, 0, 0, 0, 0)) match { // [timestamp, lat, lon, elev, value] is point's content
```

Comment:
Can explain this change?

Reply:
`get` method is deprecated, this will prevent an NPE